### PR TITLE
updated textfield.kt comments

### PR DESCRIPTION
**Pull request by GibsonRuitiari**
## Textfield.kt comments

updated textfield.kt comments

### CODE REVIEW

File: ui-components/src/main/kotlin/team/duckie/quackquack/ui/component/textfield.kt [hunk 1]
```diff
@@ -206,19 +206,19 @@ private inline fun QuackTextFieldDecorationBox(
...
         //
         // [TextField decoration items - rectangular version]
         // https://user-images.githubusercontent.com/40740128/189829922-667b1297-86fa-4e86-9db0-56de1d3ac778.png
         //
-        // However, the height of the decoration item is always the same as
-        // the height of the TextField. Therefore, it is necessary to make
-        // the width of the decoration item to wrap_content.
+        // Regardless of the shape, the height of the decoration item is always the same as
+        // the height of the TextField, as for the width,
+        // it's size is equal to the total length content of the decorated item [wrapped]
```
Comment (jisungbin) [inline ui-components/src/main/kotlin/team/duckie/quackquack/ui/component/textfield.kt:221]: Awesome explanation! You explained `wrap_content` well. But what does the "[wrapped]" at the end mean?

Comment (GibsonRuitiari) [reply ui-components/src/main/kotlin/team/duckie/quackquack/ui/component/textfield.kt:221]: wrapped means wrap_content, I should change it to wrap_content instead haha

Comment (jisungbin) [reply ui-components/src/main/kotlin/team/duckie/quackquack/ui/component/textfield.kt:221]:
```suggestion
        // it's size is equal to the total length content of the decorated item [wrap_content]
```

Cool!

Comment (GibsonRuitiari) [reply ui-components/src/main/kotlin/team/duckie/quackquack/ui/component/textfield.kt:221]: Yeap, I agree with the suggestion , it can be merged now.

**Comment by jisungbin** [comment]
Thanks for your contribution!

**Pull request merged**